crates/router_env/src/logger/types.rs

```diff
@@ -509,6 +509,8 @@ pub enum Flow {
     PaymentsPostSessionTokens,
     /// Payments start redirection flow
     PaymentStartRedirection,
+    /// Volume split for dynamic routing
+    DynamicRoutingVolumeSplit,
```

Comment:
Should we name this `VolumeSplitOnRoutingType`. This makes it seem like the volume split is on dynamic routing algorithms